discord/message.py

```diff
@@ -563,7 +563,7 @@ def system_content(self):
             else:
                 return '{0.author.name} started a call \N{EM DASH} Join the call.'.format(self)
 
-    async def delete(self):
+    async def delete(self, delay=None):
```

Comment:
```suggestion
    async def delete(self, *, delay=None):
```

This needs to be a keyword only argument. `await m.delete(10.0)` is bad readability and could be ambiguous in the future.

Reply:
Oh bugger I completely forgot. Yes, this should be that.